.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Thanks for your interest in The Odin Project. As a courtesy to our maintainers please do a search in our issues to make sure this is not a duplicate of an existing issue. In order to get issues addressed in a reasonable amount of time, we request that you include a baseline of information about the issue you're experiencing and how to reproduce it. Please provide the following:
```

Comment:
Should we create multiple issue templates? one for feature requests another for bug reports etc.

Reply:
@KevinMulhern great idea. Please review new feature request template, at your leisure.